planex/init.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+
+"""
+Creates or regenerates a Makefile with special planex-init comments
+"""
+
+import os
+import logging
+
+MAKEFILE_PATH = "/usr/share/planex"
+
+log = logging.getLogger(__name__)
+
+
+def create_makefile():
+    """ Checks if a Makefile exists with special planex-init comments in it.
+    If not, it creates or regenerates the Makefile while preserving its
+    existing contents.
+    """
+    name = "Makefile"
+
+    firstline = "# Start generated by planex-init\n"
+    autogen = "include %s/Makefile.rules\n" % (MAKEFILE_PATH)
+    endline = "# End generated by planex-init\n"
+
+    if not os.path.exists(name):
+        log.debug("Creating Makefile")
+        with open(name, 'w') as h:
+            h.write(firstline)
+            h.write(autogen)
+            h.write(endline)
+
+    with open(name, 'r') as h:
+        lines = h.readlines()
+
+    try:
+        s = lines.index(firstline)
+        e = lines.index(endline)
+
+        lines = lines[:s+1] + [autogen] + lines[e:]
+    except:
+        raise
+
+    with open(name, 'w') as h:
+        h.writelines(lines)
+
+
+def main():
+    create_makefile()
```

Comment:
Could you please add the following lines here, so this script can be run directly when testing (see configure.py for example)?
```
if __name__ == "__main__":
    main()
```